reTurn/AsyncTlsSocketBase.cxx

```diff
@@ -67,7 +67,7 @@ AsyncTlsSocketBase::connect(const std::string& address, unsigned short port)
    // into a list of endpoints.
    resip::Data service(port);
 #ifdef USE_IPV6
-   asio::ip::tcp::resolver::query query(address, service.c_str());   
+   asio::ip::tcp::resolver::query query(mSocket.lowest_layer().local_endpoint().protocol(), address, service.c_str());
```

Comment:
Remove #ifdef?  This comment likely applies to a few other places.  I'll stop making it now.  :)